pkg/pipeline/CiCdPipelineOrchestrator.go

```diff
@@ -249,9 +249,8 @@ func (impl CiCdPipelineOrchestratorImpl) PatchMaterialValue(createRequest *bean.
 		} else {
 			pipelineMaterial.CiPipelineId = createRequest.Id
 			pipelineMaterial.CreatedBy = userId
-			materialsUpdate = append(materialsUpdate, pipelineMaterial)
 			pipelineMaterial.CreatedOn = createOnTimeMap[material.GitMaterialId]
-			pipelineMaterial.CreatedBy = createByMap[material.GitMaterialId]
```

Comment:
any reason for removing this line? also we should update values of UpdatedOn, UpdatedBy to have correct audit in case needed for some debugging